docs-v2/source/includes/_tipser-script.md

```diff
@@ -243,11 +247,15 @@ Displays the full-size version of the currently selected product image.
 
 Displays the product thumbnails.
 
+<aside class="notice">
+Product Thumbnails requires fixed width and height:
```

Comment:
Zmień to proszę na "the tag with `data-tipser-modular-product-image` attribute must have fixed width and height specified"